examples/Web-AC-control/Web-AC-control.ino

```diff
@@ -73,26 +72,27 @@ WebServer server(80);
 
 bool handleFileRead(String path) {
   //  send the right file to the client (if it exists)
-  //  Serial.println("handleFileRead: " + path);
+  Serial.println("handleFileRead: " + path);
   if (path.endsWith("/")) path += "index.html";
   // If a folder is requested, send the index file
   String contentType = getContentType(path);
   // Get the MIME type
   String pathWithGz = path + ".gz";
-  if (SPIFFS.exists(pathWithGz) || SPIFFS.exists(path)) {
+  if (FILESYSTEM.exists(pathWithGz) || FILESYSTEM.exists(path)) {
     // If the file exists, either as a compressed archive, or normal
-    if (SPIFFS.exists(pathWithGz))  // If there's a compressed version available
+    // If there's a compressed version available
+    if (FILESYSTEM.exists(pathWithGz))
       path += ".gz";  // Use the compressed verion
-    File file = SPIFFS.open(path, "r");
+    File file = FILESYSTEM.open(path, "r");
     //  Open the file
     server.streamFile(file, contentType);
     //  Send it to the client
     file.close();
     // Close the file again
-    // Serial.println(String("\tSent file: ") + path);
+    Serial.println(String("\tSent file: ") + path);
```

Comment:
Ditto. You're changing the behaviour by adding in the debug messages. Is that something you mean to do?

Reply:
re-comment.